### ISSUE TITLE
Test case is not completely saved if user presses the Save button while the page is loading

### ISSUE THREAD

**Issue by cjmachado** [vertigo17/Cerberus#581]
If the user presses the Save button on the TestCase page while the page is loading, Cerberus will save the test case data that is already loaded on that page and will discard any data that is still to be loaded.
Steps to reproduce:
1. Go to TestCase page and select a case with lots of actions and properties;
2. Press the Save button on that page and wait for the page to load;
3. While loading, press the Save button again and wait for the page to fully load;
4. Cerberus will only load the data that was loaded in step 3.

**Comment by bcivel** [comment]
Save button is now displayed only after the page is loaded.

**Comment by gmoita** [comment]
And what about "Save Testcase Info" button?
Shouldn't it have the same behaviour?

**Comment by afnogueira** [comment]
@gmoita 
I have added a block loader in the page. Please check if this solution is suitable for you. 
Thank you.

**Comment by cjmachado** [comment]
I tested saving a test case using the "Save Changes" button and the "Save TestCase Info" button (at the header). I'm getting an error when clicking the "Save TestCase Info" - The user is redirected to "UpdateTestCaseWithDependencies" servlet and the following error message is shown:

Oops... Sorry but an unexpected error occurred.

Found the following messages in the log files: 

[#|2015-10-02T15:41:11.203+0200|WARNING|glassfish3.1.2|com.sun.grizzly.config.GrizzlyServiceListener|_ThreadID=113;_ThreadName=Thread-2;|More than the maximum number of request parameters (GET plus POST) for a single request ([10,000]) were detected. Any parameters beyond this limit have been ignored. To change this limit, set the maxParameterCount attribute on the Connector.|#]
